vhdbuilder/parts/k8s/cloud-init/artifacts/cis.sh

```diff
@@ -1,77 +0,0 @@
-#!/bin/bash
-
```

Comment:
keep this?

Reply:
I think we can move it, since it is just being referenced by the vhd-image-builder***.json files. And no other vhd specific file is calling the inner functions of cis.sh.
Those functions are also being used by templates_generated.go file of agentbaker. So it is safe to move it

Reply:
you're right. i thought we delete it.